IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Slave/Decoding/Int16Decoder.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace AzureIoTEdgeModbus.Slave.Decoding
+{
+    using System;
+
+    public class Int16Decoder : ValueDecoderBase, IModbusDataDecoder
+    {
+        protected override short ByteSize => 2;
+
+        protected override string ConvertToString(in Span<byte> valueBytes)
+        {
+            return BitConverter.ToInt16(valueBytes).ToString();
+        }
+
+        public short GetByteCount(short valuesToRead)
```

Comment:
This method can potentially be in base class as virtual method so it can be overridden if needed by any decoder but mostly remains as-is.

Reply:
Some styling corrections in editorconfig like use of 'this' is not present but's a minor thing.

Reply:
Moved method to base class (public virtual) and changed to this.ByteSize.